### PR TITLE
set lower and upper limit for partitions

### PR DESCRIPTION
**Pull request by kim-lindhard-dfds**

### CODE REVIEW

File: src/CapabilityService.WebApi/Infrastructure/Api/ExceptionToStatusCode.cs [hunk 1]
```diff
@@ -19,6 +20,8 @@ public static ActionResult Convert(Exception exception)
 				case DFDS.CapabilityService.WebApi.Features.Kafka.Domain.Exceptions.TopicAlreadyExistException _:
 				case DFDS.CapabilityService.WebApi.Domain.Exceptions.TopicAlreadyExistException _:
 					return new ConflictObjectResult(new {exception.Message});
+				case PartitionsCountNotAllowedException _:
+					return new UnprocessableEntityObjectResult(new {exception.Message});
 				default:
 					return null;
```
Comment (wcarlsen) [inline src/CapabilityService.WebApi/Infrastructure/Api/ExceptionToStatusCode.cs:34]: null result isn't sufficient

**Review by wcarlsen — CHANGES_REQUESTED** [review]
Again with the case switches man ;-)

**Comment by kim-lindhard-dfds** [comment]
[//]: # (4744e426-5795-11e9-8647-d663bd873d93)

|| **CodeScene Delta Analysis Results** |
| --- | ---- |
| **Risk** |5|
| **Quality Gates** |Fail|
| **Description** |The change is medium risk, adds 150 lines, deletes 66 lines of code in 6 files, which is more than your typical change set. The risk is lower since it is a very experienced author with many contributions.|
| **Warnings** |Delta Analysis Info<br/><ul><li>2 files in this change set did not exist with the same name on the main branch. This is probably because those files have been renamemd. Skipping code health checks for those files.</li></ul><br/>Degrades in Code Health<br/><ul><li>CapabilityController.cs degrades from a Code Health of 9 -> 8.67</li></ul>|
| **Code Health Delta Descriptions:** |CapabilityController.cs<br/><ul><li> **Degradations:** <ul><li>Code Duplication - getting worse</li></ul></li></ul>|

**Review by wcarlsen — APPROVED** [review]
Great man. Good work

**Comment by kim-lindhard-dfds** [comment]
[//]: # (4744e426-5795-11e9-8647-d663bd873d93)

|| **CodeScene Delta Analysis Results** |
| --- | ---- |
| **Risk** |5|
| **Quality Gates** |Fail|
| **Description** |The change is medium risk, adds 150 lines, deletes 66 lines of code in 6 files, which is more than your typical change set. The risk is lower since it is a very experienced author with many contributions.|
| **Warnings** |Delta Analysis Info<br/><ul><li>2 files in this change set did not exist with the same name on the main branch. This is probably because those files have been renamemd. Skipping code health checks for those files.</li></ul><br/>Degrades in Code Health<br/><ul><li>CapabilityController.cs degrades from a Code Health of 9 -> 8.67</li></ul>|
| **Code Health Delta Descriptions:** |CapabilityController.cs<br/><ul><li> **Degradations:** <ul><li>Code Duplication - getting worse</li></ul></li></ul>|

**Pull request merged**